daffodil-japi/src/test/java/org/apache/daffodil/example/TestJavaAPI.java

```diff
@@ -969,4 +969,52 @@ public void testJavaAPI18() throws IOException, ClassNotFoundException {
       assertEquals("over", outputter.getResult().getRootElement().getText());
     }
 
+  /*
+   * This test uses the same input file as the CLI blob test, test_blob_backtracking.
+   * The necessary file may need to be generated, see instructions in:
+   *
+   * daffodil-cli/src/it/scala/org/apache/daffodil/blob/TestBlob.scala
+   *
+   * If it has already been generated, it can be copied from:
+   *
+   * daffodil/daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/2049MB.bin
+   *
+   * to:
+   *
+   * daffodil/daffodil-sapi/src/test/resources/test/sapi
+   *
+   */
+    @Test
+    public void testJavaAPI19() throws IOException, ClassNotFoundException {
+      // Demonstrate that we cannot use the API to continue a parse with an invalid InputSource
+      // ie. after a runtime SDE. This test needs to be run with an input file larger than 256MB
+      org.apache.daffodil.japi.Compiler c = Daffodil.compiler();
+      c.setValidateDFDLSchemas(false);
+      java.io.File schemaFile = getResource("/test/japi/blob_backtracking.dfdl.xsd");
+      c.setDistinguishedRootNode("root", null);
+      ProcessorFactory pf = c.compileFile(schemaFile);
```

Comment:
Looks like this test is failing here. I think blob_backtracking.dfdl.xsd need to be added?

Reply:
Yup, forgot to add it, also I forgot to comment it out.